browser/ui/sidebar/sidebar_browsertest.cc

```diff
@@ -521,24 +521,27 @@ IN_PROC_BROWSER_TEST_F(SidebarBrowserTest, PrefsMigrationTest) {
                   ->IsDefaultValue());
 }
 
-IN_PROC_BROWSER_TEST_F(SidebarBrowserTest, PRE_SidePanelResizeTest) {
```

Comment:
Didn't know about `PRE_` prefix. 👍

Reply:
Yeah, it's spanning restart :)  - https://www.chromium.org/developers/testing/browser-tests/#tests-spanning-restarts

Reply:
I haven't used it yet, but maybe `PRE_PRE_A`, `PRE_A`, and `A` would work :)